src/content/reference/react-dom/server/renderToStaticNodeStream.md

```diff
@@ -37,6 +37,9 @@ The stream will produce non-interactive HTML output of your React components.
 
 * `reactNode`: A React node you want to render to HTML. For example, a JSX element like `<Page />`.
 
+* **optional** `options`: An object for server render.
+  * **optional** `identifierPrefix`: A string prefix React uses for IDs generated by [`useId`.](/reference/react/useId) Useful to avoid conflicts when using multiple roots on the same page. Must be the same prefix as passed to [`hydrateRoot`.](/reference/react-dom/client/hydrateRoot#parameters)
```

Comment:
```suggestion
  * **optional** `identifierPrefix`: A string prefix React uses for IDs generated by [`useId`.](/reference/react/useId) Useful to avoid conflicts when using multiple roots on the same page.
```
because its content cannot be hydrated